### PR TITLE
Fixed error for if quality control variable has no stand name.

### PR DESCRIPTION
**Pull request by fxmzb123**

### CODE REVIEW

File: compliance_checker/imos/imos.py [hunk 1]
```diff
@@ -1560,10 +1560,10 @@ def check_quality_control_variable_standard_name(self, ds):
             for data_variable in self._data_variables:
                 ancillary_variables = find_ancillary_variables_by_variable(ds.dataset, data_variable)
                 if qc_variable in ancillary_variables:
-                    value = getattr(data_variable, 'standard_name') + ' ' + 'status_flag'
+                    value = getattr(data_variable, 'standard_name', '') + ' ' + 'status_flag'
                     result_name = ('var', 'quality_variable', qc_variable.name, data_variable.name, 'check_standard_name')
-                    if getattr(qc_variable, 'standard_name') != value:
-                        reasoning = ["Standard is not corrent"]
+                    if getattr(qc_variable, 'standard_name', '') != value:
+                        reasoning = ["Standard name is not corrent"]
```
Comment (lbesnard) [inline compliance_checker/imos/imos.py]: could you please correct the typo ?cheers

**Comment by fxmzb123** [comment]
@lbesnard the typo has been fixed.

**Comment by mhidas** [comment]
@fxmzb123 GitHub says this one can't be automatically merged now that I've merged the tide-up branch.

**Comment by mhidas** [comment]
@fxmzb123 This needs a rebase.

**Comment by fxmzb123** [comment]
@mhidas I rebased this branch and it can be merged automatically

**Comment by mhidas** [comment]
Thanks @fxmzb123 !

**Pull request merged**